server.py

```diff
@@ -208,18 +289,45 @@ def contest(code):
     return bottle.template("contest.html", contest=contest, questions=contest.questions)
 
 
+
 @app.get("/question/<path:path>")
 def download(path):
+    """
+    Downloads the question to local system
+
+    : param path : path to download from
+    :return : Static Files
+    """
+
     return bottle.static_file(path, root=question_dir)
 
 
+
 @app.get("/static/<filepath:path>")
 def server_static(filepath):
+    """
+    Static files for server
+
+    : param path : path to download from
+    :return : Static Files
+    """ 
     return bottle.static_file(filepath, root=os.path.join(dir_path, "static"))
 
 
 @app.get("/ranking/<code>")
 def contest_ranking(code):
+    """
```

Comment:
nothing new. Perhaps if we had multiple rankings then it would have made sense to add a docstring here. IDK. Right now this docstring repeats what the code does.

Reply:
I think here i am explaing how contest ranking are created , i thin it would the reader to understand it more.